File: tests/Contexts/Rutine/Exercises/domain/ExerciseBarMother.ts
import { ExerciseBar } from '../../../../../src/Contexts/Rutine/Exercises/domain/ExerciseBar';
import { IntegerMother } from '../../../Shared/domain/IntegerMother';

export class ExerciseBarMother {
  static create(value: number): ExerciseBar {
    return new ExerciseBar(value);
  }

  static random(): ExerciseBar {
    return this.create(IntegerMother.random());
  }
}

File: src/test/java/com/kuro/rutineweb/exercises/entities/ExerciseBarMother.java
package com.kuro.rutineweb.exercises.entities;

import com.kuro.rutineweb.Shared.entities.IntMother;

public class ExerciseBarMother {
    public static ExerciseBar create(int value) {
        return new ExerciseBar(value);
    }

    public static ExerciseBar random() {
        return new ExerciseBar(IntMother.random(20));
    }
}
